docs/calicoctl/profile.md

```diff
@@ -315,6 +321,11 @@ calicoctl profile <PROFILE> rule add (inbound|outbound) [--at=<POSITION>]
     <ICMPCODE>: Specific code number related to ICMP type.
 ```
 
+Note: If the `<SRCCIDR>` and `<DSTCIDR>` are both specified, the IP version 
```

Comment:
I would simplify this by saying not to mix IPv4 and IPv6 addresses or protocols in a single rule.